src/main/java/org/springframework/samples/petclinic/rest/JacksonCustomVisitDeserializer.java

```diff
@@ -72,6 +78,10 @@ public Visit deserialize(JsonParser parser, DeserializationContext context)	thro
 		visit.setDate(visitDate);
 		visit.setDescription(description);
 		visit.setPet(pet);
+        visit.setVet(vet);
```

Comment:
We can use a utility method for copying same properties. For example, since we already use Spring in the project, we can directly go for BeanUtils.copyProperties(...)